Scripts/Runtime/Jobs/Collections/CollectionAccessController.cs

```diff
@@ -161,39 +154,45 @@ public JobHandle Acquire(AccessType accessType)
         }
 
         /// <summary>
-        /// Allows the <see cref="CollectionAccessController{TContext,TKey}"/> to be aware of the work that you are doing
+        /// Allows the <see cref="CollectionAccessController{TKey}"/> to be aware of the work that you are doing
         /// for a specific <see cref="AccessType"/>. You must call <see cref="Release"/> after any call to <see cref="Acquire"/>
         /// before you call <see cref="Acquire"/> again.
         /// </summary>
-        /// <param name="releaseAccessDependency">The <see cref="JobHandle"/> to the job that is doing the reading or
-        /// writing from/to the underlying collection this <see cref="CollectionAccessController{TContext,TKey}"/>
-        /// is gating access to.</param>
+        /// <param name="releaseAccessDependency">
+        /// The <see cref="JobHandle"/> to the job that is doing the reading or
+        /// writing from/to the underlying collection this <see cref="CollectionAccessController{TKey}"/>
+        /// is gating access to.
+        /// </param>
         public void Release(JobHandle releaseAccessDependency)
         {
             Debug.Assert(!IsDisposed);
             ValidateReleaseState(releaseAccessDependency);
 
             switch (m_State)
             {
-                case AcquisitionState.Disposing:
                 case AcquisitionState.ExclusiveWrite:
                     //If you were exclusively writing, then no one else can do any writing or reading until you're done
-                    //In the disposing case, we set to the same dependencies but gate in the checks
-                    m_ExclusiveWriteDependency = JobHandle.CombineDependencies(m_ExclusiveWriteDependency, releaseAccessDependency);
-                    m_SharedWriteDependency = m_ExclusiveWriteDependency;
-                    m_SharedReadDependency = m_ExclusiveWriteDependency;
+                    m_ExclusiveWriteDependency
+                        = m_SharedWriteDependency
+                            = m_SharedReadDependency
+                                = releaseAccessDependency;
```

Comment:
Is this how Rider formats these lines? I'd expect just one indent for the whole assignment chain.

Reply:
Yeah this is what its doing. I tried toggling on and off a whole bunch of the indent settings but I couldn't find one that changes this. 

I don't particularly mind this formatted this way. I think it calls out the multi-assignment nicely. 

Definitely open to changing it if I can figure out which setting it is.

Reply:
Not going to die on this hill! It is what it is.